src/library_openal.js

```diff
@@ -532,12 +532,12 @@ var LibraryOpenAL = {
       // a displacement from the listener.
       if (src.relative) {
         // Negate the listener direction since forward is -Z.
-        var lBackX = -listener._direction[0];
-        var lBackY = -listener._direction[1];
-        var lBackZ = -listener._direction[2];
-        var lUpX = listener._up[0];
-        var lUpY = listener._up[1];
-        var lUpZ = listener._up[2];
+        var lBackX = listener.direction[0];
```

Comment:
why did the `-` negations go away?

Reply:
My mistake, must have been too fast at removing the underscores from member names. Thanks for spotting this, I'll fix it.